docs/modules/operation/pages/provisioning/detectors/BgpSessionDetector.adoc

```diff
@@ -0,0 +1,22 @@
+
+= BGP Session Detector
+
+The BGP Service Detector can be used for detecting if a BGP-Session to a peering partner (peer-ip) is functional on devices with SNMP enabled. It will make a query against the RFC1269 MIB (`.1.3.6.1.2.1.15.3.1`) over SNMP to check a peering partner's status.
+It checks that the return value is in the valid range of 1 (idle) to 6 (established).
```

Comment:
Maybe an additional hint to make sure the detector figures out if a peering configuration exists ignoring the fact if it's really working or not, which is the job of the monitor.